src/index.html

```diff
@@ -29,5 +30,6 @@
       s.parentNode.insertBefore(wf, s);
     })();
   </script>
+  <script src="https://cdn.auth0.com/js/lock-8.min.js"></script>
```

Comment:
Not without adding a few other build-pipeline dependencies (and this authentication approach is not the only one you'd ever use so it'd just populate the dependencies list with modules that we don't use).